### DISCUSSION TITLE
Including prompts in GPT descriptions

### DISCUSSION THREAD

**Discussion by brobbins2001** [polywock/Awesome-GPTs#14] [General]
I'm curious about what others here think of including your prompt in the GPT description. I chose to include mine, but I'm wondering what potential downsides/upsides others see. For me (adventure master GPT) I mainly chose to share it because my GPT is meant to follow a lot of arbitrary rules, that may be helpful for the user to know.

**Comment by brobbins2001** [comment]
Perhaps they shouldn't be put in the file created by "anon" but rather in a discussion post like this. This would allow GPT creators to update the instructions here without the need for a pull request.

**Comment by polywock** [comment]
Negatives 
1. Easy for bad faith actors to just re-create your GPT for potential profit. You spent a year fine tuning this, yet, someone could just copy in 10 seconds. 
2. Easy for players to re-create and customize your GPT for their own playing style. You might miss out potential revenue (depending on how the GPT store pans out). It would also make it easy for users to just make their version public and potentially compete. 

Positives 
1. Collaborative development. In that case, you should create a new Github repository where users can contribute. 

Let me know if you want to remove it, I would recommend doing so.

**Reply by brobbins2001** [reply]
I agree with this, I do however find it unlikely that OpenAI will ever allow us to make money off of our GPTs, but in the event they do It may be worthwhile. I'll put in a PR to remove it.

**Reply by brobbins2001** [reply]
If a GPT creator wants to include further instructions than were automatically gathered, should it be okay for the creator to create a PR with modifications to the description file, or is there another way you'd prefer to handle this?

**Reply by polywock** [reply]
I use tooling to generate the support/ files, so letting me know through a Github issue will be my preferred method.

**Reply by brobbins2001** [reply]
Sounds good, thanks for the clarification
